File: recursion/java/FibonacciSeries.java
public class FibonacciSeries {

    public int fibonacciSeries(int n){
        if (n <= 0){
            return 0;
        } else if (n == 1 || n == 2){
            return 1;
        }
        int previousSum = fibonacciSeries(n-1)+fibonacciSeries(n-2);
        return previousSum;
    }
    public static void main(String[] args){
        FibonacciSeries fs = new FibonacciSeries();
        int n = 6;
        int sum = fs.fibonacciSeries(n);
        System.out.println("Fibonacci of "+n+" is : "+sum);
    }
}

File: recursion/python/FibonacciSeries.py
def fibonacci_series(n):
    if n <= 0:
        return 0
    elif n == 1 or n == 2:
        return 1
    
    previous_sum = fibonacci_series(n-1) + fibonacci_series(n-2)
    return previous_sum

n = 6
sum = fibonacci_series(n)
print("Fibonacci of ",n," is : ",sum)
